pkg/parser/additional/certificate_utils.go

```diff
@@ -0,0 +1,79 @@
+package additional
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+// CheckCertificateBody verifies if the attribute 'certificate_body' refers a file
+func CheckCertificateBody(content string) (b bool, s string) {
+	var re = regexp.MustCompile(`[0-9a-zA-Z-/\\_]+\.pem`)
+	match := re.FindAllString(content, -1)
+
+	if match == nil {
+		return false, ""
+	}
+
+	return true, match[0]
```

Comment:
If you want only the first match, why no use [FindString](https://golang.org/pkg/regexp/#Regexp.FindString) instead `FindAllString`.

Also, instead of returning `(bool, string)`, I think you could return just string and when string is empty is the same as false... So you could just return the matched string

Reply:
Thank you so much for your suggestions, Felipe 😄